src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    int number;
```

Comment:
Consider making the `number` field private to adhere to the principle of encapsulation. This will prevent direct access to the field from outside the class.